.yarn/patches/react-native-web-webview-npm-1.0.2-2a7da84b81.patch

```diff
@@ -0,0 +1,40 @@
+diff --git a/dist/index.js b/dist/index.js
```

Comment:
@johnsmith-gooddollar @sirpy
basically this patch is just so that I could continue implementing the widget.
vite does not natively allow .html imports/require

I tried a couple of suggested solutions, but at runtime it still fails to load.
I commented out this 'postMock.html' for now through patch, but I am not sure what it does or what now is not working?

Reply:
fixed

Reply:
thanks